_utilities.scss

```diff
@@ -1,3 +1,11 @@
+// Tool Utilities
+// Tools have corresponding mixins
+@import "tools/antialiased/utility";
```

Comment:
It's a little odd/confusing to have `tools` has the prefix for files being called into utilities. Would it make sense to re-prefix these with `utilities`?

Reply:
tools is the folder they are in, not a prefix. They are classes that tap directly into a tool (function or mixin) unlike the other utilities.

Reply:
Systems has a similar situation, but gets pulled in within the _tools.scss file.

Reply:
Maybe a more detailed comment about their placement would help alleviate confusion about why they're in the `utilities` file? Something like what you have in the PR description

Reply:
🙌